cookies/models.py

```diff
@@ -616,6 +623,27 @@ class GilesUpload(models.Model):
     )
     state  = models.CharField(max_length=2, choices=STATES)
 
+    PRIORITY_HIGH = 10
+    PRIORITY_MEDIUM = 20
+    PRIORITY_LOW = 30
```

Comment:
Is there a reason the priorities are the opposite of their name (low has the highest number)? It's a little odd and I think it's more likely to add a priority "highest" later rather than 'lowest'.